starfish/test/test_multiprocessing.py

```diff
@@ -0,0 +1,113 @@
+"""
+These tests ensure that we can share an np-array or something that encapsulates an np-array among
+processes.  The data is shared such that any process can write to the array and it is visible to all
+the other processes.  It is up to the developer that uses this mechanism to ensure data integrity is
```

Comment:
Any suggestions about how a developer would ensure this? Could be useful to include here.

Reply:
I can't think of any general guidelines to ensure this. :(

Reply:
Ok. No problem.